modes/dictation_mode.talon

```diff
@@ -65,6 +65,11 @@ formatted <user.format_text>:
 ^format selection <user.formatters>$:
     user.formatters_reformat_selection(formatters)
 
+# Numbers
+numb <user.number_string>: "{number_string}"
+numb <user.number_string> colon <user.number_string>: "{number_string_1}:{number_string_2}"
+numb <user.number_string> (dot | point) <digit_string>: "{number_string}.{digit_string}"
```

Comment:
1. Any reason not just include `prose_number` inside `raw_prose`?
2. Failing that, any reason not to do `<user.prose_number>: "{prose_number}"`?

Reply:
1. Silly me! I see no reason not to.
2. I think it's helpful to support both numerals and numbers as words.

Reply:
great, if it gets included in `raw_prose` then these commands aren't needed (iiuc?) - they'll be handled by the `<user.raw_prose>: auto_insert(raw_prose)` command.

Reply:
Yep. I'll make that change later.